regression-tests.dnsdist/test_TLS.py

```diff
@@ -536,20 +539,67 @@ class TestTLSTicketsKeyAddedCallback(DNSDistTest):
     newServer{address="127.0.0.1:%s"}
     addTLSLocal("127.0.0.1:%s", "%s", "%s", { provider="openssl" })
 
-    callbackCalled = 0
+    lastKey = ""
+    lastKeyLen = 0
+
     function keyAddedCallback(key, keyLen)
-      callbackCalled = keyLen
+      lastKey = key
+      lastKeyLen = keyLen
     end
+    setTicketsKeyAddedHook(keyAddedCallback)
+    """
+
+    def testSetTicketsKey(self):
+        """
+        TLSTicketsKey: test setting new key and the key added hook
+        """
 
+        newKey = ''.join(random.choice(string.ascii_uppercase + string.digits) for _ in range(80))
+        print("about to send command: `{}`".format("getTLSFrontend(0):setTicketsKey(\"{}\")".format(newKey)))
```

Comment:
The comment is wrong (likely because the function got renamed since ;)), but I guess it can go altogether?